Error: Cannot find module 'rollup/bin/rollup'

Trying to build the project, got an error after running gulp build, what am I doing wrong?

>  node --version
> v14.15.5
>  npm --version
> 7.5.4
>  gulp --version
> CLI version: 2.3.0
> Local version: 4.0.2

Steps to reproduce
> git clone https://github.com/nagix/chartjs-plugin-colorschemes
> cd chartjs-plugin-colorschemes
> npm install
> gulp build

```
isangjae@siin-iui-MacBookPro chartjs-plugin-colorschemes % gulp build
[14:26:46] Using gulpfile /Volumes/DK_DEV/project/chartjs-plugin-colorschemes/gulpfile.js
[14:26:46] Starting 'build'...
[14:26:46] 'build' errored after 3.24 ms
[14:26:46] Error: Cannot find module 'rollup/bin/rollup'
Require stack:
- /Volumes/DK_DEV/project/chartjs-plugin-colorschemes/gulpfile.js
- /usr/local/lib/node_modules/gulp/node_modules/gulp-cli/lib/shared/require-or-import.js
- /usr/local/lib/node_modules/gulp/node_modules/gulp-cli/lib/versioned/^4.0.0/index.js
/* global Promise */
- /usr/local/lib/node_modules/gulp/node_modules/gulp-cli/index.js
- /usr/local/lib/node_modules/gulp/bin/gulp.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.resolve (internal/modules/cjs/helpers.js:94:19)
    at /Volumes/DK_DEV/project/chartjs-plugin-colorschemes/gulpfile.js:24:21
    at new Promise (<anonymous>)
    at run (/Volumes/DK_DEV/project/chartjs-plugin-colorschemes/gulpfile.js:22:9)
    at /Volumes/DK_DEV/project/chartjs-plugin-colorschemes/gulpfile.js:40:9
    at build (/Volumes/DK_DEV/project/chartjs-plugin-colorschemes/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (domain.js:413:15)
    at runBound (domain.js:424:12)
    at asyncRunner (/Volumes/DK_DEV/project/chartjs-plugin-colorschemes/node_modules/async-done/index.js:55:18)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,10 +4,13 @@ node_js:
 
 before_script:
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
+
+# Syntax replaces old method of invoking xvfb directly
+services:
+  - xvfb
 
 script:
   - gulp test --coverage

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -37,7 +37,7 @@ function run(bin, args) {
 }
 
 gulp.task('build', function() {
-	return run('rollup/bin/rollup', ['-c', argv.watch ? '--watch' : '']);
+	return run('rollup/dist/bin/rollup', ['-c', argv.watch ? '--watch' : '']);
 });
 
 gulp.task('test', function(done) {

--- a/src/plugins/plugin.colorschemes.js
+++ b/src/plugins/plugin.colorschemes.js
@@ -11,7 +11,7 @@ var EXPANDO_KEY = '$colorschemes';
 
 // pluginBase snippet fixes the chartjs 3 incompatibility, and is backwards-compatible
 // by Github user gebrits (https://github.com/gebrits/chartjs-plugin-colorschemes)
-// 
+//
 // Chartjs 2 => Chart.defaults.global
 // Chartjs 3 => Chart.defaults
 const pluginBase = Chart.defaults.global || Chart.defaults;
@@ -52,8 +52,10 @@ var ColorSchemesPlugin = {
 	beforeUpdate: function(chart, args, options) {
 		// Please note that in v3, the args argument was added. It was not used before it was added,
 		// so we just check if it is not actually our options object
-		if (options === undefined) options = args
-		
+		if (options === undefined) {
+			options = args;
+		}
+
 		var scheme = getScheme(options.scheme);
 		var fillAlpha = options.fillAlpha;
 		var reverse = options.reverse;

--- a/src/plugins/plugin.colorschemes.js
+++ b/src/plugins/plugin.colorschemes.js
@@ -9,7 +9,13 @@ var hoverReset = Chart.DatasetController.prototype.removeHoverStyle.length === 2
 
 var EXPANDO_KEY = '$colorschemes';
 
-Chart.defaults.global.plugins.colorschemes = {
+// pluginBase snippet fixes the chartjs 3 incompatibility, and is backwards-compatible
+// by Github user gebrits (https://github.com/gebrits/chartjs-plugin-colorschemes)
+// 
+// Chartjs 2 => Chart.defaults.global
+// Chartjs 3 => Chart.defaults
+const pluginBase = Chart.defaults.global || Chart.defaults;
+pluginBase.plugins.colorschemes = {
 	scheme: 'brewer.Paired12',
 	fillAlpha: 0.5,
 	reverse: false,
@@ -43,7 +49,11 @@ function getScheme(scheme) {
 var ColorSchemesPlugin = {
 	id: 'colorschemes',
 
-	beforeUpdate: function(chart, options) {
+	beforeUpdate: function(chart, args, options) {
+		// Please note that in v3, the args argument was added. It was not used before it was added,
+		// so we just check if it is not actually our options object
+		if (options === undefined) options = args
+		
 		var scheme = getScheme(options.scheme);
 		var fillAlpha = options.fillAlpha;
 		var reverse = options.reverse;

